app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java

```diff
@@ -194,7 +195,8 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
     }
 
     @OnClick(R.id.tvNoOfImages)
-    public void onClick() { }
+    public void onClick() {
```

Comment:
why this change?

Reply:
oops, sorry. before committing I had done "reformat code" which had added space between parentheses and moved the brace to the new line. I have reverted the code style the way it was before.